galasa-ui/messages/en.json

```diff
@@ -359,5 +359,26 @@
     "remove": "Remove",
     "cancel": "Cancel",
     "save": "Save"
+  },
+  "TableDesignContent": {
+    "description": "Customize your table view by showing, hiding, and reordering columns.",
```

Comment:
I think "your table view" should be clarified a bit to make it clear that the table design tab customizes the table in the results tab - maybe something like "Customize the test run results table by showing, hiding, and reordering columns."? Feel free to come up with something better!